content/docs/user-guide/basic-concepts.md

```diff
@@ -0,0 +1,34 @@
+# Basic Concepts of DVC
+
+DVC streamlines large data files and binary models into a single Git
+environment. This approach will not require storing binary files in your Git
+repository.
+
+![](/img/flow-large.png) _DVC data management_
```

Comment:
let's omit this, not sure this is needed here to be honest

Reply:
OK, removed img in a93f24f2.